main.go

```diff
@@ -132,6 +135,58 @@ func sendMetric(v []string, t map[string]string, message string, tags [][]string
 	fmt.Fprintf(conn, put)
 }
 
+func rejectMetric(app string, metric string, metricType string) {
+	appParts := strings.SplitN(app, "-", 2)
+	appName := appParts[0]
+	appSpace := appParts[1]
+
+	rejectMessage := struct {
+		Log        string      `json:"log"`
+		Stream     string      `json:"stream"`
+		Time       time.Time   `json:"time"`
+		Kubernetes interface{} `json:"kubernetes"`
+		Topic      string      `json:"topic"`
+	}{
+		"Unique metrics limit exceeded. Metric discarded: [" + metricType + "] " + metric,
+		"stdout",
+		time.Now(),
+		struct {
+			PodName       string `json:"pod_name"`
+			ContainerName string `json:"container_name"`
+		}{
+			"akkeris-warning",
+			appName,
+		},
+		appSpace,
+	}
+
+	var logshuttleURL string
+	if os.Getenv("LOGSHUTTLE_URL") != "" {
+		logshuttleURL = os.Getenv("LOGSHUTTLE_URL")
+	} else {
+		logshuttleURL = "http://logshuttle.akkeris-system.svc.cluster.local"
```

Comment:
If the logshuttle url isn't set,  should this functionality just be turned off entirely? That way the metrics syslog collector could be used without the logshuttle, if we require the logshuttle it might lower who may end up using this outside of akkeris.

Worth some thought, i could be swung either way. If you want to keep this as is, that's fine.

Reply:
Good question. I think there's a bit of work that can be done overall to make it more platform-agnostic - for example, I'll take it one step further and add the ability to disable the entire metric limiting functionality as well as the logshuttle functionality

Reply:
Added `ENABLE_UNIQUE_METRIC_LIMIT` to control the metric limiting feature
`LOGSHUTTLE_URL` now controls whether or not rejection messages are sent